docs/DeveloperGuide.md

```diff
@@ -1,34 +1,1135 @@
+
+- Alternative 2: Saving during any alterations made to the Diet session
+
+|     |     |
+|-----|-----|
+|**Pros** | The files will still be saved even if a crash occurs|  
+|**Cons** | Saving often might be taxing on the user's computer|
+#### 4.3.5. Delete a previously created diet session: `delete`
+
+The feature allows users to delete previously created diet sessions.
+
+**Implementation**  
+When the user types `delete [diet session ID]` from a Diet manager instance the following sequence occurs. 
+1. The user keys in `delete 1`.
+    
+    1. A `DietSessionUi` component will call `dietSessionUI.getInput()`. 
+    1. The input is then parsed in `processCommand()` which splits the input into a command portion and the input parameters.   
+    
+2. Creation of `DietSessionDelete()` command object from input
+    1. The `CommandLib` is referenced to find information on `DietSessionDelete()`.
+    2. A `DietSessionDelete()` command class instantiation is created and the `execute()` method is called.
+    
+3. Executing Command
+    1. `DietSessionDelete()` will then delete the diet session at index `1` based on the `list` command.
+
+The sequence diagram below summarizes how Diet sessions are deleted:
+
+![Delete_Diet_Session_Sequence_Diagram](pictures/Zeon/DietSessionDelete.png)
+#### 4.3.6. Edit a past diet session: `clear`
+
+The feature allows users to clear all previously created diet sessions at once.
+
+**Implementation**  
+When the user types `clear` the following sequence occurs. 
+1. The user keys in `clear`.
+    
+    1. A `DietSessionUi` component will call `dietSessionUI.getInput()`. 
+    1. Input will be parsed in `processCommand()`.   
+    
+2. Creation of `DietSessionClear()` command object from input
+    1. The `CommandLib` is referenced to find information on `DietSessionClear()`.
+    2. A `DietSessionClear()` command class instantiation is created and the `execute()` method is called.
+    
+3. Executing Command
+    1. This will iterate through every file in saves/diet/ and delete it.
+
+The sequence diagram below summarizes how Diet sessions are all cleared:
+
+![Delete_Diet_Session_Sequence_Diagram](pictures/Zeon/DietSessionClear.png)
+#### 4.3.7. Exit the Diet manager: `end`
+
+The function returns the user back to the main menu of The Schwarzenegger.
+
+**Implementation**  
+When the user types `end` the following sequence occurs. 
+1. The user keys in `end`.
+    
+    1. A `DietSessionUi` component will call `dietSessionUI.getInput()`. 
+    1. Input will be parsed in `processCommand()`.  
+    
+2. Exiting of inputLoop()
+    The inputLoop() exits when userInput.equals("end"), returning to the `Start()` method, then ending the `DietManager` instance.
+
+
+[&#8593; Return to Top](#developer-guide)
+
+
+### 4.4. <a id="workout-related-features">Workout-related Features</a>
+#### 4.4.1. <a id="creating-a-new-workout-session">Creating a New Workout Session</a>
```

Comment:
![Screenshot 2020-10-29 at 12 44 26 PM](https://user-images.githubusercontent.com/69417796/97526625-7f112500-19e4-11eb-9881-e9d390c7ed34.png)

Is this the correct way to interpret "create a new WorkoutSession"?